source/reference/api/reverse.txt

```diff
@@ -0,0 +1,84 @@
+.. _c2c-api-reverse:
+
+===========
+``reverse``
+===========
+
+.. include:: /includes/preview-warning
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Description
+-----------
+
+Reverses the direction of a committed sync operation.
+
+For example, if you have a ``COMMITTED`` a sync operation where
```

Comment:
Consider reformatting:

For example:
You have committed a sync operation

cluster0 is the source
cluster1 is the destination
To sync new writes from cluster1 to cluster0, use the reverse endpoint, {+c2c-product-name+} .

Reply:
Tried reformatting this a bit.